app/session_helper.rb

```diff
@@ -0,0 +1,9 @@
+module SessionHelper
+
```

Comment:
I feel 'weird' about this being a 'Helper', but I can't say exactly why. I'm used to helpers being used in both Views and Controllers (mostly Views), and `create_session` feels v controller-specific